cli.go

```diff
@@ -361,5 +361,8 @@ func (i *CLI) amqpErrorWatcher(amqpConn *amqp.Connection) {
 	if ok {
 		i.logger.WithField("err", err).Error("amqp connection errored, terminating")
 		i.cancel()
+		time.Sleep(2 * time.Minute)
```

Comment:
2 might be too long, I just was saying that because it was a little underneath half the alert threshold for missing queue consumers.